install-launchd.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+set -e
+
+if [ "$UID" -eq 0 ] ; then
+    echo "$0 is not meant to be run as root" >&2
+    exit 1
+fi
+
+calcheck="$HOME/Library/LaunchAgents/com.calcheck.timer.plist"
+dir=$(pwd)
+stdout="$dir/log.out"
+stderr="$dir/log.err"
+
+if [ -e $calcheck ]; then
+    echo "Existing plist file found. Unloading $calcheck"
+    launchctl unload $calcheck
+fi
```

Comment:
Can you do this with launchctl instead?  I would expect launchctl allows it somehow.

Reply:
not that i'm aware of. if you try to load and there's an existing service, it will throw up. looking at the launchctl load help, there's no option to force override.